index.html

```diff
@@ -12,56 +12,56 @@
-                    <span class="navbar-toggler-icon"></span>
-                </button>
-                <div class="collapse navbar-collapse" id="navbarNavDropdown">
-                    <ul class="navbar-nav">
-                        <li class="nav-item "><a href="http://127.0.0.1:5500"><img id="logo" src="img/icons/logo.png"
-                                    alt="logo"></a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link active" aria-current="page" href="index.html">Home</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link active" aria-current="page" href="about.html">About</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link active" href="#">Program</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link active" href="#">Join</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link active" href="#">Sponsor</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link active" href="#">News</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link active" href="#">Join!</a>
-                        </li>
-                    </ul>
-                </div>
-            </div>
-        </nav>
-    </header>
     <main>
+        <header>
+            <div id="social">
+                <ul class="d-flex flex-row justify-content-md-end">
+                    <li><img id="fb" src="img/icons/fb.png" alt="facebook">
+                    </li>
+                    <li><img id="tt" src="img/icons/twitter.png" alt="twitter"></li>
+                    <li class="litext">English</li>
+                    <li class="litext">My Page</li>
+                    <li class="litext">Logout</li>
+                </ul>
+            </div>
+            <nav class="navbar navbar-expand-md navbar-light">
+                <div class="container-fluid">
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                        data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false"
+                        aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                        <ul class="navbar-nav">
+                            <li class="nav-item "><a href="http://127.0.0.1:5500"><img id="logo"
+                                        src="img/icons/logo.png" alt="logo"></a>
+                            </li>
```

Comment:
- [ ] There should be links from `Home` page to `About` page and the opposite too. The link back to the `Home` Page in the live version does not work. An error page is being displayed. Please, let's fix it. Hint: `http://127.0.0.1:5500` in your `a` tag is the source of the error.